src/sanitizer.js

```diff
@@ -66,6 +64,7 @@ const SELF_CLOSING_TAGS = {
   'col': true,
   'hr': true,
   'img': true,
+  'input': true,
```

Comment:
Are there any other I missed? I'd rather have a full list there, even if it's blacklisted tags.